### ISSUE TITLE
Feedback Widget unaccessible by keyboard

### ISSUE THREAD

**Issue by wackerow** [ethereum/ethereum-org-website#6517]
**Describe the bug**
Tab stops aren't working for the feedback widget, and aria-labels missing for screen readers. 

**To Reproduce**

1. Go to https://ethereum.org
2. Attempt to navigate to (and through) FeedbackWidget with just the keyboard

**Expected behavior**
Should be able to tab to the FeedbackWidget "dot" and navigate easily with keyboard within this modal when it opens.
Should be compatible with screen readers.

**Additional context**
PR incoming
